Sources/TuistCore/Models/Config.swift

```diff
@@ -11,6 +11,7 @@ public struct Config: Equatable, Hashable {
         case xcodeProjectName(String)
         case organizationName(String)
         case disableAutogeneratedSchemes
+        case disableAutogeneratedNamespace
```

Comment:
I find the name `namespace` not accurate enough because it doesn't have a connection with "resources". What about calling it, `disableSynthesizedResourceAccessors`. I know it's long, but I think it reflects better what the feature is about.

Reply:
Sure, that is, indeed, a better name, will do!